swagger.yaml

```diff
@@ -1,83 +1,192 @@
 swagger: '2.0'
+
 info:
-  title: Web API Skeleton
-  description: Skeleton for Dropwizard Web APIs
-  version: '0.0'
-  license:
-    name: GNU Affero General Public License Version 3
-    url: http://www.gnu.org/licenses/agpl-3.0.en.html
-externalDocs:
-  description: GitHub Repository
-  url: https://github.com/osu-mist/web-api-skeleton
-host: localhost
-schemes:
-  - http
-basePath: /api/v0
-consumes:
-  - text/plain
-produces:
-  - text/plain
+  title: Music Collection API
+  version: "0.1"
+  
+#consumes: [ 'application/json' ]
+produces: [ 'application/json' ]
+
 paths:
-  /:
+  '/shelf':
+    post:
```

Comment:
Your POST /shelf api endpoint should support the same json schema as the GET /shelf/{id} except for the id property since that's assigned by the api endpoint .